### PR TITLE
feat(instance): add console command

### PR DESCRIPTION
**Pull request by Sh4d1**

### CODE REVIEW

File: internal/gotty/client.go [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+	"syscall"
+	"time"
+
+	"github.com/containerd/console"
+	"github.com/gorilla/websocket"
+	"github.com/scaleway/scaleway-sdk-go/scw"
+)
+
+// Scaleway GoTTY websocket endpoint
+var wsURLs = map[scw.Zone]string{
```
Comment (Sh4d1) [inline internal/gotty/client.go:18]: Not sure which is better between `tty.scaleway.com` and `tty-<region>.scaleway.com` :thinking:

File: internal/gotty/client.go [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+	err = cns.SetRaw()
+	if err != nil {
+		return fmt.Errorf("error setting raw terminal: %w", err)
+	}
+	defer cns.Reset()
+	defer cns.Close()
+
+	// Create a chanel that will receive all resizes signals
+	resizeChan := make(chan os.Signal, 1)
+	signal.Notify(resizeChan, syscall.SIGWINCH)
```
Comment (Sh4d1) [inline internal/gotty/client.go:91]: We should split the syscall in another `_unix.go` file and a dummy one in a `_windows.go` file with the righ build tags

File: internal/gotty/client.go (hunk 1 above)
Comment (jerome-quere) [reply internal/gotty/client.go:18]: tty.scaleway.com do not work for AMS

Comment (Sh4d1) [reply internal/gotty/client.go:18]: :scream: :heavy_check_mark:

File: internal/gotty/client.go [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+	// Create a chanel that will receive all resizes signals
+	resizeChan := make(chan os.Signal, 1)
+	unsubscribe := subscribeToResize(resizeChan)
+	defer unsubscribe()
+
+	wsChan, wsErrChan := websocketReader(conn)
+	cnsChan, cnsErrChan := consoleReader(cns)
+
+	// Force first resize
+	resizeChan <- syscall.SIGWINCH
```
Comment (Sh4d1) [inline internal/gotty/client.go:97]: same as above, need to be in another file

**Pull request merged**